contracts/DEX/SimpleDex.sol

```diff
@@ -38,7 +39,15 @@ contract SimpleDex {
         address maker
     ) internal returns (uint256) {
         //Create unfilled order
-        orders[orderCounter] = Types.Order(maker, amount, price, 0, side, expiration, block.timestamp);
+        Types.Order storage order = orders[orderCounter];
+        order.maker = maker;
+        order.amount = amount;
+        order.price = price;
+        order.side = side;
+        order.expiration = expiration;
+        order.creation = block.timestamp;
```

Comment:
Was this change for forward compatibility with solc 0.7?

Reply:
I was running it on Solidity 0.6.0, I'll find us the latest stable OZ version to run and get a ticket up for that migration